src/txHashBuilder.c

```diff
@@ -1163,6 +1167,8 @@ static void txHashBuilder_assertCanLeaveMint(tx_hash_builder_t* builder)
 
 	switch (builder->state) {
 	case TX_HASH_BUILDER_IN_MINT:
+		ASSERT(builder->multiassetData.remainingAssetGroups == 0);
+		ASSERT(builder->multiassetData.remainingTokens == 0);
```

Comment:
Hasn't this been done in another PR?... I don't see a reason to make this change when adding script data hash to tx body.

Reply:
I am adding this small change (moving asserts into this place) to `multisig-final-update-to-be-tested-yet`, we want to use them in multisig-final.

Reply:
Well, this was moved with the alonso changes that introduced multiple steps after mint. But yeah, can be moved.